### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by alefranco1**
This commit will implement a developer environment and Webpack configuration with a working seeding script that will populate a Mongo database when it is run within the root directory in the terminal. Create a database connection and initiate a node serve with a working API call to GET a single document from the Mongodb.

### CODE REVIEW

**Comment by bastipacudan** [comment]
@alefranco1 good work the only comment that I have is you should use a more specific endpoint for your get request and also I noticed that you already merged it to master, I suggest we follow Jordan's tip that the reviewer should be the one to merge or give the go signal for it.

File: Server/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const app = express();
+const port = 3000;
+
+const Camps = require('../database-mongodb/Camp.js')
```
Comment (jordanstholmes) [inline Server/index.js:5]: @alefranco1 @bastipacudan require statements should go at the top of the file.

File: Server/index.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+app.get('/:campsiteId', (req, res) => {
+  console.log(req.params)
```
Comment (jordanstholmes) [inline Server/index.js:8]: @alefranco1 @bastipacudan Try not to leave stray console logs in code that will be merged into master.

File: Server/index.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const app = express();
+const port = 3000;
+
+const Camps = require('../database-mongodb/Camp.js')
+
+app.get('/:campsiteId', (req, res) => {
```
Comment (jordanstholmes) [inline Server/index.js:7]: @alefranco1 @bastipacudan resource path should have some descriptive noun before the id: `'/campsites/:siteId'`

File: Server/index.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  console.log(req.params)
+    Camps.find({ newId: req.params.campsiteId}, (err, camps)=> {
+        if (err){
+          res.send(err)
```
Comment (jordanstholmes) [inline Server/index.js:11]: @alefranco1 @bastipacudan Nice job on doing something with the error :) Be sure to send it with the appropriate status code.

File: Server/index.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+        }else {
```
Comment (jordanstholmes) [inline Server/index.js:12]: @alefranco1 @bastipacudan spacing (linting)

Comment (jordanstholmes) [reply Server/index.js:12]: @alefranco1 @bastipacudan Also missing semi-colons lines 13, 11, 8, 5. If you don't already have it setup, I suggest using the eslint extension in vs-code along with the AirBnB style guide config (AirBnB style guide is an FEC and SDC requirement)

File: database-mongodb/index.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
+const database = mongoose.connect('mongodb://localhost/seed');
```
Comment (jordanstholmes) [inline database-mongodb/index.js:2]: @alefranco1 @bastipacudan Consider using a different database name (e.g., 'campsites' or something)

File: database-mongodb/seed.js [hunk 7]
```diff
@@ -0,0 +1,26 @@
+const database = require('./index.js');
+const Camp = require('./Camp.js');
+const faker = require('faker');
+
+var sampleData = [];
+for (var i = 0; i < 100; i++) {
+    var sampleObject = {
+        newId: i,
+        host: faker.name.findName(),
+        hostImage: faker.image.avatar(),
```
Comment (jordanstholmes) [inline database-mongodb/seed.js:10]: @alefranco1 @bastipacudan For FEC, images should be hosted on S3

File: database-mongodb/seed.js [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+            image: faker.image.avatar(), 
+            votes: faker.random.boolean()
+        }]
+     };
+     sampleData.push(sampleObject);
+};
+
+const insertSampleCamps = function() {
+    Camp.create(sampleData)
+      .then(() => database.disconnect()).catch((err) => console.log(err));
```
Comment (jordanstholmes) [inline database-mongodb/seed.js:23]: @alefranco1 @bastipacudan `.catch` should go on the next line.

File: database-mongodb/seed.js [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+  };
+  
+insertSampleCamps();
```
Comment (jordanstholmes) [inline database-mongodb/seed.js:26]: @alefranco1 @bastipacudan File should end with a newline character (see red arrow above). AirBnB style guide linting should point this out. The newline character helps certain command line tools display files in a readable fashion.

File: webpack.config.js [hunk 10]
```diff
@@ -15,7 +15,7 @@ module.exports = {
         include: SRC_DIR,
         loader: 'babel-loader',
         query: {
-          presets: ['react', 'es2015']
+          presets: ['@babel/preset-react', 'airbnb', '@babel/preset-env']
```
Comment (jordanstholmes) [inline webpack.config.js:18]: @alefranco1 @bastipacudan I don't think `airbnb` belongs there

**Pull request merged**